Swig bindings for R call fprintf but fail to include stdio.h.

Reported upstream as issue #555:
  https://github.com/swig/swig/issues/555
diff -ru swig-3.0.5/Lib/r/rrun.swg swig-3.0.5.new/Lib/r/rrun.swg
--- swig-3.0.5/Lib/r/rrun.swg	2015-01-31 09:51:17.000000000 -0800
+++ swig-3.0.5/Lib/r/rrun.swg	2015-10-23 14:03:31.679583617 -0700
@@ -20,6 +20,7 @@
 
 #include <Rdefines.h>
 #include <Rversion.h>
+#include <stdio.h>
 #include <stdlib.h>
 #include <assert.h>
 
